cluster/src/main/java/org/apache/zeppelin/interpreter/SnappydataZeppelinInterpreter.java

```diff
@@ -0,0 +1,800 @@
+/*
```

Comment:
Check the licensing info. I think both Apache and SnappyData license would be applicable here.